setup.py

```diff
@@ -22,6 +22,15 @@
 this_dir = os.path.dirname(os.path.abspath(__file__))
 torch_dir = torch.__path__[0]
 
+
+def hipBLASlt_supported():
+    supported_arch = ['gfx942']
+    device_props = torch.cuda.get_device_properties(0); 
```

Comment:
This fails in situations where apex is being built as part of a Dockerfile, since GPUs are not visible at that stage.